src/index.ts

```diff
@@ -415,6 +432,87 @@ export class FlashbotsBundleProvider extends providers.JsonRpcProvider {
     }
   }
 
+  /**
+   * Method to send a carrier tx into the public mempool
+   *
+   * @param bundle  FlashbotsBundle with AT LEAST signed bundled transactions in signedBundledTransactions field obtained
+   *  from {@link signBundle} method, and blockTarget.
+   * @param validatorPublicKey  The public key of the validator that will be able to decrypt the bundle and include it
+   *  into the bundle pool.
+   * @param signer  Signer who will sign the carrier transaction.
+   * @param carrierTx TransactionRequest whose data field will carry the encrypted bundle : MAY be an incomplete
+   *  object which will be populated with default values.
+   *
+   * @return Promise<TransactionResponse> Promise containing the response for the carrier tx
+   * */
+
+  public async publish(
```

Comment:
non-intuitive func name

Reply:
will come up with better alternatives for func names